tests/integration/jobs/tasks.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('tasks', function () {
       expect(response.status).toBe(httpStatusCodes.OK);
       expect(taskFindMock).toHaveBeenCalledTimes(1);
       expect(taskFindMock).toHaveBeenCalledWith({
-        jobId: jobId,
```

Comment:
Please add test in tasks - (if there isn't already) to get with parameters and withaout

Reply:
Added